src/components/NewDraftWizard/WizardComponents/WizardFooter.js

```diff
@@ -40,44 +55,43 @@ const WizardFooter = ({ nextButtonRef }: nextButtonRefProp) => {
   const dispatch = useDispatch()
   const wizardStep = useSelector(state => state.wizardStep)
   return (
-    <div className={classes.footerRow}>
-      <div>
-        <Link component={RouterLink} aria-label="Cancel adding a new draft" to="/">
-          <Button
-            variant="contained"
-            color="secondary"
-            className={classes.cancelButton}
-            onClick={() => dispatch(reset())}
-          >
-            Cancel
-          </Button>
-        </Link>
-        {wizardStep >= 1 && (
-          <Button variant="contained" color="primary" onClick={() => dispatch(decrement())}>
-            Back
-          </Button>
+    <div>
+      <div className={classes.phantom} />
+      <div className={classes.footerRow}>
+        <div>
+          <Link component={RouterLink} aria-label="Cancel creating a new submission" to="/">
+            <Button
+              variant="contained"
+              color="secondary"
+              onClick={() => dispatch(reset())}
+              className={classes.footerButton}
+            >
+              Cancel
+            </Button>
+          </Link>
+        </div>
+        {wizardStep >= 0 && (
+          <div>
+            <Button
+              variant="contained"
+              color="primary"
+              disabled={wizardStep >= 1 ? false : true}
+              className={classes.footerButton}
+              onClick={async () => {
+                await nextButtonRef.current.submitForm()
```

Comment:
Note for later: this should probably be something else than form submission? maybe post folder content to backend,

Reply:
this was just part of UI, functionality maybe later.